mixins/kubernetes/rules/recording_and_alerting_rules/templates/k8s-resource-windows-cluster-rules-group-2.json

```diff
@@ -17,7 +17,7 @@
                 "rules": [
                     {
                         "record": "windows_pod_container_available_2",
-                        "expression": "windows_container_available{job = \"windows-exporter\"} * on(container_id) group_left(container, pod, namespace, cluster) max(kube_pod_container_info{job = \"kube-state-metrics\"}) by(container, container_id, pod, namespace, cluster)"
```

Comment:
@bragi92 - are these files (under mixins/kubernetes/rules/recording_and_alerting_rules/templates) used anywhere ?

Reply:
You're right. They're not used anymore. I had created them for the private preview of the windows recording rules and dashboards. I will create a PR to delete them.

Reply:
ok thanks Kaveesh.